media_driver/linux/Xe_M/ddi/media_libva_caps_dg2.cpp

```diff
@@ -916,7 +927,18 @@ VAStatus MediaLibvaCapsDG2::CreateEncAttributes(
     if (IsHevcProfile(profile))
     {
         attrib.type = (VAConfigAttribType) VAConfigAttribPredictionDirection;
-        attrib.value = VA_PREDICTION_DIRECTION_PREVIOUS | VA_PREDICTION_DIRECTION_FUTURE | VA_PREDICTION_DIRECTION_BI_NOT_EMPTY;
+        if (!IsHevcSccProfile(profile))
+        {
+            attrib.value = VA_PREDICTION_DIRECTION_PREVIOUS | VA_PREDICTION_DIRECTION_FUTURE | VA_PREDICTION_DIRECTION_BI_NOT_EMPTY;
```

Comment:
not support P frame?

Reply:
Gen12 doesn't support P frame, corrected the change.